pennylane/debugging.py

```diff
@@ -298,10 +505,45 @@ def _measure(measurement):
 
 
 def tape():
-    """Access the quantum tape of the circuit.
+    """Access the tape of the quantum circuit.
```

Comment:
Should we add a "see also" qml.tape?

Reply:
I think it might cause more confusion, since they are used in very different contexts.